modP2p/src/org/aion/p2p/P2pConstant.java

```diff
@@ -48,5 +48,20 @@
     // write queue timeout
     WRITE_MSG_TIMEOUT = 5000,
 
-    BACKWARD_SYNC_STEP = 128;
+    REQUEST_SIZE = 24,
+
+    /** @implNote must be strictly larger than {@link #REQUEST_SIZE} */
+    LARGE_REQUEST_SIZE = 25,
```

Comment:
can we just define REQUEST_SIZE + 1?

Reply:
My intention is for this value to be significantly larger. Tests with 60 looked promising, but more experiments are needed to determine the potential drawbacks of such a large value and what would be an optimal constant. For now, I updated this value to 40 (1c6bf7b) and opened an issue for further experimentation (#648).